src/main/java/Duke.java

```diff
@@ -1,10 +1,117 @@
+import DukePackage.ChatUI;
+import DukePackage.Parser;
+import DukePackage.Storage;
+import DukePackage.Task;
+import DukePackage.TaskType;
+import java.util.Objects;
+
+
+/**
+ * The Duke class is the main class that runs the Duke chatbot application.
+ */
 public class Duke {
+
+    /**
+     * The main method that starts the Duke application.
+     * @param args The command-line arguments passed to the application.
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        // standard response
+        String horizontalLine = "   ------------------------------------------------------------------------";
+        String noDescError = "     ☹ OOPS!!! The description of a todo cannot be empty.";
+        String noCommandError = "     ☹ OOPS!!! I'm sorry, but I don't know what that means :-(";
+
+        // initialise
+        Storage storage = new Storage();
+        Parser parser = new Parser();
+        ChatUI ui = new ChatUI();
+
+        // read from txt file and create tasks and put into storage
+        storage.loadListFromFile();
+        ui.printIntro();
+
+        while (true) {
+            String input = parser.getInput();
+            String[] parts = input.split(" ");
+            ui.printDivider();
+
+            switch (parts[0]) {
```

Comment:
Hey, I may be wrong, but according to checkstyle, the indentations for the case lines are supposed to be the same as that of switch, while the codes for each case is supposed to be one indentation more than the case statements.